src/operator/nn/mkldnn/mkldnn_pooling-inl.h

```diff
@@ -113,20 +113,15 @@ inline bool SupportMKLDNNPooling(const PoolingParam &param,
   if (!ret)
     return false;
 
-  if (param.pooling_convention == pool_enum::kValid)
+  if (param.pooling_convention == pool_enum::kValid) {
     return true;
-  else
-    return false;
-
-// need to support pooling convention full
-// https://issues.apache.org/jira/browse/MXNET-33
-#if 0
-  if (((dshape[2] + 2 * param.pad[0] - param.kernel[0]) % param.stride[0] == 0) &&
-      ((dshape[3] + 2 * param.pad[1] - param.kernel[1]) % param.stride[1] == 0))
-    return true;
-  else
-    return false;
-#endif
+  } else {
+    // currently, only max-pooling is supported for full convention
+    if (param.pool_type == pool_enum::kMaxPooling)
+      return true;
+    else
+      return false;
```

Comment:
`return param.pool_type == pool_enum::kMaxPooling`

Reply:
Fixed. Thanks for review.  :)